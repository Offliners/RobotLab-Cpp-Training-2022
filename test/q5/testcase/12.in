(16/9*3-15+13-20+14/14*9/3-2-(1/5))/2=*(15/18-7/9/2-(1-10)*17+5*1/3*7/(16*17))-((5/5)+3*9*16/19+12+19)=+(19*12)=/8-6-16-10+19/11-11-12*9=-9*18*4-16*20-3/2-7=/13*4/9/18+8-(3/19*20+19)-((6*5*8)/3)=*20-1+7-17+(6+9*4*18+10)/6*9-12/18/2+15+13+7-(18/17)+7=
