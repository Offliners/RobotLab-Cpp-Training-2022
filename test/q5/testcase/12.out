387
234
0
0
